Orchestrator/src/main/java/com/tainavbot/server/orchestrator/domain/enumeration/HealthStatusEnum.java

```diff
@@ -0,0 +1,6 @@
+package com.tainavbot.server.orchestrator.domain.enumeration;
+
+public enum HealthStatusEnum {
+    OK,
+    DEAD
+}
```

Comment:
Duplicate

Reply:
Needs to be enum per layer of hexagonal infrastructure